xray_vision/mask/manual_mask.py

```diff
@@ -56,7 +56,7 @@
 
 class ManualMask(object):
     @ensure_ax_meth
-    def __init__(self, ax, image, cmap='gray', undo_history_depth=20):
+    def __init__(self, ax, image, cmap='gray', vmax=None, undo_history_depth=20):
```

Comment:
add an `imshow_kwargs` input which lets you pass an arbitrary dict through to imshow.